src/Astreiko.Homework6/Program.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+
+namespace Astreiko.Homework6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var currentAtm = new Atm();
+
+            currentAtm.AddBalanceHandler += CurrentAtm_AddBalanceHandler;
```

Comment:
Events usually named as a verb in passive or with On prefix to make it more describing and readable